lib/common/procfs.c

```diff
@@ -29,16 +29,15 @@
  * \param[out] name     If not NULL, a char[16] to hold the process name
  * \param[out] pid      If not NULL, will be set to process ID of entry
  *
- * \return 0 on success, -1 if entry is not for a process or info not found
- *
+ * \return Standard Pacemaker return code
```

Comment:
I wonder if it'd be possible to link the "Standard Pacemaker return code" text to the definition of pcmk_rc_e.  I know we don't build docs for internal functions (though I would kind of like to have them for my own reference), and we're looking at moving away from publican, but it might be a nice-to-have for the future.

Reply:
The API docs are actually generated by doxygen rather than publican, so it may be possible. Don't know the right way to do it off the top of my head though. You can also generate internal API docs locally by setting INTERNAL_DOCS to YES in Doxyfile.in and running "make -C doc doxygen", though I haven't tried that.